modules/op-improving-performance.adoc

```diff
@@ -0,0 +1,19 @@
+// This module is included in the following assemblies:
+// * resource/managing-pipelines-performance.adoc
+
+:_content-type: PROCEDURE
+[id="improving-performance_{context}"]
+= Improving {pipelines-shortname} performance
+
+If you experience slowness or recurrent failures of pipeline runs, you can take any of the following steps to improve the performance of {pipelines-shortname}.
+
+* Increase the number of nodes in the {OCP} cluster on which {pipelines-shortname} runs.
++
+[NOTE]
+====
+Increasing the CPU and RAM of individual nodes does not affect performance significantly.
```

Comment:
i'm not sure if we can make this statement - there are times when node resources can become a performance bottleneck and can increase the workload failure rate - at that point increasing these resources will directly improve the performance